packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.4
```

Comment:
Usually a minor internal refactor would just be a bugfix version, which for <1.0 is `+N` by Dart convention, so I would make this 0.3.3+1. (I did an effectively minor version bump for the Pigeon changes since they made significant changes; even that was borderline IMO.)

Reply:
Fixed,